bitnami/keycloak/26/debian-12/rootfs/opt/bitnami/scripts/keycloak-env.sh

```diff
@@ -179,22 +185,28 @@ export KEYCLOAK_ROOT_LOG_LEVEL="${KEYCLOAK_ROOT_LOG_LEVEL:-INFO}"
 export KEYCLOAK_PROXY_HEADERS="${KEYCLOAK_PROXY_HEADERS:-}"
 export KEYCLOAK_PRODUCTION="${KEYCLOAK_PRODUCTION:-false}"
 export KEYCLOAK_EXTRA_ARGS_PREPENDED="${KEYCLOAK_EXTRA_ARGS_PREPENDED:-}"
+KEYCLOAK_DATABASE_VENDOR="${KEYCLOAK_DATABASE_VENDOR:-"${KC_DB:-}"}"
 export KEYCLOAK_DATABASE_VENDOR="${KEYCLOAK_DATABASE_VENDOR:-postgresql}"
-KEYCLOAK_DATABASE_HOST="${KEYCLOAK_DATABASE_HOST:-"${DB_ADDR:-}"}"
+KEYCLOAK_DATABASE_HOST="${KEYCLOAK_DATABASE_HOST:-"${KC_DB_URL_HOST:-}"}"
 export KEYCLOAK_DATABASE_HOST="${KEYCLOAK_DATABASE_HOST:-postgresql}"
-KEYCLOAK_DATABASE_PORT="${KEYCLOAK_DATABASE_PORT:-"${DB_PORT:-}"}"
+KEYCLOAK_DATABASE_PORT="${KEYCLOAK_DATABASE_PORT:-"${KC_DB_URL_PORT:-}"}"
 export KEYCLOAK_DATABASE_PORT="${KEYCLOAK_DATABASE_PORT:-5432}"
-KEYCLOAK_DATABASE_USER="${KEYCLOAK_DATABASE_USER:-"${DB_USER:-}"}"
+KEYCLOAK_DATABASE_USER="${KEYCLOAK_DATABASE_USER:-"${KC_DB_USERNAME:-}"}"
 export KEYCLOAK_DATABASE_USER="${KEYCLOAK_DATABASE_USER:-bn_keycloak}"
-KEYCLOAK_DATABASE_NAME="${KEYCLOAK_DATABASE_NAME:-"${DB_DATABASE:-}"}"
+KEYCLOAK_DATABASE_NAME="${KEYCLOAK_DATABASE_NAME:-"${KC_DB_URL_DATABASE:-}"}"
 export KEYCLOAK_DATABASE_NAME="${KEYCLOAK_DATABASE_NAME:-bitnami_keycloak}"
-KEYCLOAK_DATABASE_PASSWORD="${KEYCLOAK_DATABASE_PASSWORD:-"${DB_PASSWORD:-}"}"
+KEYCLOAK_DATABASE_PASSWORD="${KEYCLOAK_DATABASE_PASSWORD:-"${KC_DB_PASSWORD:-}"}"
 export KEYCLOAK_DATABASE_PASSWORD="${KEYCLOAK_DATABASE_PASSWORD:-}"
-KEYCLOAK_DATABASE_SCHEMA="${KEYCLOAK_DATABASE_SCHEMA:-"${DB_SCHEMA:-}"}"
+KEYCLOAK_DATABASE_SCHEMA="${KEYCLOAK_DATABASE_SCHEMA:-"${KC_DB_SCHEMA:-}"}"
 export KEYCLOAK_DATABASE_SCHEMA="${KEYCLOAK_DATABASE_SCHEMA:-public}"
-KEYCLOAK_JDBC_PARAMS="${KEYCLOAK_JDBC_PARAMS:-"${JDBC_PARAMS:-}"}"
+KEYCLOAK_JDBC_PARAMS="${KEYCLOAK_JDBC_PARAMS:-"${KC_DB_URL_PROPERTIES:-}"}"
 export KEYCLOAK_JDBC_PARAMS="${KEYCLOAK_JDBC_PARAMS:-}"
+KEYCLOAK_JDBC_DRIVER="${KEYCLOAK_JDBC_DRIVER:-"${KC_DB_DRIVER:-}"}"
 export KEYCLOAK_JDBC_DRIVER="${KEYCLOAK_JDBC_DRIVER:-postgresql}"
+export KC_DB_POOL_INITIAL_SIZE="${KC_DB_POOL_INITIAL_SIZE:-}"
+export KC_DB_POOL_MAX_SIZE="${KC_DB_POOL_MAX_SIZE:-}"
+export KC_DB_POOL_MIN_SIZE="${KC_DB_POOL_MIN_SIZE:-}"
+export KC_DB_URL="${KC_DB_URL:-}"
```

Comment:
Thanks for adding these new env vars. However, they are not used right now in the code so you need to change the initialization scripts to take them into account. You will need to [configure_database method](https://github.com/bitnami/containers/blob/main/bitnami/keycloak/26/debian-12/rootfs/opt/bitnami/scripts/libkeycloak.sh#L130) to achieve it.